FRINXio/frinx-frontend
# FR-310 lsp count

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FR-310` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |


HOW TO TEST:

test with https://github.com/FRINXio/frinx-inventory-server/pull/463

you need to have recent version of topology-discovery
got to mpls layer - you should see incoming/outcoming numbers on edge when click on the node
